js/components/app.tsx

```diff
@@ -1,21 +1,62 @@
+import { reload } from "../browser";
 import { Help } from "./help";
 import { Home } from "./home";
 import { List } from "./operatorSignIn/list";
-import { ReactElement } from "react";
-import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import { captureException } from "@sentry/react";
+import { ReactElement, useEffect } from "react";
+import {
+  createBrowserRouter,
+  RouterProvider,
+  useRouteError,
+} from "react-router-dom";
+
+const ErrorBoundary = (): ReactElement => {
+  const error = useRouteError() as Error;
```

Comment:
**suggestion:** I see that this just returns `unknown` in the types provided by the package, which is unfortunate. It might be possible to gate the `captureException` call behind an `if (error instanceof Error)` condition or something, but then we probably also want to do something if it's not an instance of `Error` because that's itself a weird case we want visibility into.

Reply:
`captureException(error)` worked elsewhere when it was `unknown`, so I think the only unsafe use here is when displaying `error.name/message/stack` in the UI. So maybe the `if (error instanceof Error)` check can go down there, and we show something less useful if we can't figure out what type `error` is.

Reply:
👍 let me know what you both think about 5c0c72 ?

Reply:
I like it!